Add watch-only wallets to common use cases (#1136)
BitcoinDesign/Guide: guide/designing-products/common-user-flows.md

Mixing coins is a technique to improve transaction privacy by making a special transaction. Multiple senders participate in the same transaction by sending and receiving identical amounts of bitcoin. Because this intertwines the transaction history of those coins, it becomes much harder to trace the individual coins and their respective owners.

> One of these technique is called Payjoin, which you can learn more about [here]({{ '/guide/case-studies/payjoin/' | relative_url }}).

Reviewer: ```suggestion
One of these techniques is called Payjoin, which allows users (the sender and the receiver) to mix their coins during the course of a regular transaction. Learn more about payjoin [here]({{ '/guide/case-studies/payjoin/' | relative_url }}).
```

</div>

## Watching a wallet

<div class="center" markdown="1">

{% include image.html
   image = "/assets/images/guide/designing-products/common-user-flows/watch.jpg"
   retina = "/assets/images/guide/designing-products/common-user-flows/watch@2x.jpg"
   alt-text = ""
   width = 100
   height = 100
   layout = "float-left"
%}

> A watch-only wallet allows users to monitor the balance and transactions of a bitcoin wallet without having the private keys required to spend the funds. Setup involves importing an address or the [extended public key]({{ '/guide/glossary/#extended-public-key-xpub-ypub-zpub' | relative_url }}) (xpub) of the wallet they want to monitor. This provides read-only access to the wallet's transaction history and current balance.

Reviewer 2: Looks great, I just want to add one thing, more from practice, unsire if it's relevant for this, but if a watch-only wallet is generated on the daily spending wallet itself, where the key is then removed from the device, it's essential to communicate that to the user, but also ensure recovery of a seed phrase has been done, otherwise, we've seen in practice that a lot of users, generate a watch-only wallet, it gets removed from BTCPay Server, they never backed it up (since we don't have an extra step to confirm seed phrase backup) and keep receiving money to that wallet... You can imagine a frustration they have once they relise funds are irrecoverable. 

I guess here it's assume that watch only wallet is externally imported, but there are wallets that I guess generate watch-only as well.

Author: Interesting, I have not heard of applications that generate watch-only wallets.

This is a very general page that loosely describes the different things people do in wallets, so it's not necessarily the place to dive into the finer details. If we added a more detailed page to the savings wallet, I'd make sure to include it.

What's your recommendation? Add a note here? Or not?

Reviewer 2: No recommendation, the page lgtm, just wanted to mention it, perhaps you're right and watch only should never be generated just imported.

</div>

## Watching a wallet

<div class="center" markdown="1">

{% include image.html
   image = "/assets/images/guide/designing-products/common-user-flows/watch.jpg"
   retina = "/assets/images/guide/designing-products/common-user-flows/watch@2x.jpg"
   alt-text = ""
   width = 100
   height = 100
   layout = "float-left"
%}

> A watch-only wallet allows users to monitor the balance and transactions of a bitcoin wallet without having the private keys required to spend the funds. This can be useful for tracking purposes or for keeping an eye on a wallet's activity without risking the security of the funds.

Reviewer: - The ability to derive receive addresses should be mentioned, since that is what allows the tracking/monitoring to be done, but also is useful in itself for sharing with others
- Both sentences in this para seem to be saying the same thing in different ways (I do this a lot myself :/)

</div>

> To create a watch-only wallet, users typically import an address or the [extended public key]({{ '/guide/glossary/#extended-public-key-xpub-ypub-zpub' | relative_url }}) (xpub) of the wallet they want to monitor. This provides read-only access to the wallet's transaction history and current balance.

Reviewer: - I don't understand the "import an address" part
- not sure a watch-only wallet is ever "created", since it's basically importing an existing wallet but without the private keys...probably much of the tricky part is semantic

```suggestion
To set up a watch-only wallet, users typically import the [extended public key]({{ '/guide/glossary/#extended-public-key-xpub-ypub-zpub' | relative_url }}) (xpub) of the wallet they want to monitor. This provides read-only access to the wallet's transaction history and current balance.
```
